auth-foundation/src/main/java/com/okta/authfoundation/credential/Credential.kt

```diff
@@ -490,7 +527,7 @@ private sealed class CredentialState {
         override val tags: Map<String, String>
     ) : CredentialState()
 
-    object Deleted : CredentialState()
+    data object Deleted : CredentialState()
```

Comment:
Changing object to data object doesn't do anything meaningful for our use case, but it's good practice to use it for sealed classes (and for suppressing IDE lint in this case)